manifest.json

```diff
@@ -7,24 +7,32 @@
   "content_scripts": [
     {
       "matches": ["<all_urls>"],
-      "js": ["web3/index.js", "web3/inpage.js"],
+      "js": ["content/loader.js"],
       "run_at": "document_start"
-    },
-    {
-      "matches": ["<all_urls>"],
-      "js": ["content/index.js"]
     }
   ],
   "web_accessible_resources": [
+    "content/index.js",
```

Comment:
Thoes files doesn't need to be `web_accessible_resources`, why do we need to add them here?